lib/constants.js

```diff
@@ -40,6 +40,9 @@ module.exports = Object.freeze({
     UNPROCESSABLE_ENTITY: 422,
     BAD_REQUEST: 400
   },
+  HTTP_TIMEOUT: {
```

Comment:
Rename this to `HTTP_REQUEST` as it then makes it generic and in future you can house other props under it.

Reply:
ok..